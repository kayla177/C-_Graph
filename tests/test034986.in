ENTITY I3Y9RHAQ name_I3Y9 conference
ENTITY DX16SOSW name_DX16 author
ENTITY QGP2H0SE name_QGP2 author
RELATIONSHIP I3Y9RHAQ published_in DX16SOSW 3.2
PRINT I3Y9RHAQ
HIGHEST
EXIT
