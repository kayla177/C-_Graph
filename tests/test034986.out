success
success
success
success
DX16SOSW
I3Y9RHAQ DX16SOSW 3.2
